ansible/templates/covid_act_now-params-prod.json.j2

```diff
@@ -0,0 +1,12 @@
+{
+  "static_file_dir": "./static",
+  "export_dir": "/common/covidcast/receiving/covid_act_now",
+  "cache_dir": "./cache",
+  "wip_signal": true,
```

Comment:
Oops this is my fault for providing an outdated params.json, but the `wip_signal` key is no longer needed.
Sorry about that!

Reply:
Makes sense. I'll remove it.